crates/bevy_pbr/src/render/light.rs

```diff
@@ -1543,6 +1553,18 @@ pub fn prepare_lights(
 
                 view_lights.push(view_light_entity);
 
+                // If this light is using occlusion culling, add the appropriate components.
+                if light.occlusion_culling {
+                    commands
+                        .entity(view_light_entity)
+                        .insert(OcclusionCulling)
```

Comment:
You know that insert accepts a tuple of components?

Reply:
Yeah, just old bad habit :) Fixed.